trackers/react-native-tracker/src/subject.ts

```diff
@@ -0,0 +1,87 @@
+import { CorePluginConfiguration, PayloadBuilder, TrackerCore } from '@snowplow/tracker-core';
+import { ScreenSize, SubjectConfiguration } from './types';
+
+export function newSubject(core: TrackerCore, configuration?: SubjectConfiguration) {
```

Comment:
I would however like an explicit return type of sorts here.

Reply:
I agree that we should have well defined return types for publicly exported functions, but is it important for internal functions as well? I feel like it's more of a nuisance than useful for internal code as we don't need to specify a public contract for this code. It'd also create a lot of boilerplate here that I'm not sure if it's worth it.